PlanMyTrip/PlanMyTrip/View Controllers/ResultsViewController.m

```diff
@@ -358,4 +312,17 @@ -(void)searchCarResultsDictionary: (NSDictionary *)carSearchInformation returnAr
         }
     }
 }
+
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    if([segue.identifier isEqualToString:@"confToDisplay"]){
+        
+        DisplayResultsViewController *resultsView = [segue destinationViewController];
+        resultsView.flightResults = self.flightResults;
+        resultsView.hotelResults = self.hotelResults;
+        //resultsView.carResults = (NSString *)self.carResults;
```

Comment:
forget to clean here?

Reply:
The car results are delayed sometimes due to a completion block not being executed in time. The RS works but sometimes it doesn't so I commented it out when I was doing my demo. It is one of the things I am currently working on to fix.